pegasus/sites.v3/code.org/public/js/redirectTeacherDashboard.js

```diff
@@ -0,0 +1,41 @@
+var script = document.querySelector("script[data-teacherdashboard]");
+var scriptData = JSON.parse(script.dataset.teacherdashboard);
+var studioUrlPrefix = scriptData.studio_url;
+
+/**
+ * Maps paths that *do not* match between pegasus and dashboard teacher dashboards.
+ * The key should be the path (prefaced with a forward slash) in pegasus, and the value
+ * should be the corresponding path in dashboard.
+ */
+var urlMap = {
+  "/print_signin_cards": "/login_info",
+  "/manage": "/manage_students"
+};
+
+function redirectToDashboard() {
```

Comment:
I can see you've been careful to avoid using any es6 in this file, so I think this will all work. However, we are probably going to need to keep this file around for quite some time (old urls die hard), so it would really be better for it to live within our apps build system, in `apps/src/sites/code.org/pages/public/teacher-dashboard/index.js`. Happy to discuss in case I'm missing a tradeoff or obstacle here.

Reply:
nope, i think that makes a lot of sense. i originally made a separate .js file since i wasn't quite ready to delete `teacher-dashboard/index.js`, but now i think we are 🎉